### ISSUE TITLE
[Draft] Community Bounty #9 - "Repo/Documentation Improvements"

### ISSUE THREAD

**Issue by bwhm** [Joystream/community-repo#87]
### Tags
- `Research`
- `Testing`
- `Proofreading`
- `Coding`

## Description
As the repos are updated, it's difficult to avoid introducing broken links, errors with images, grammar mistakes, formatting errors, etc. in the README.md files. This makes it difficult to navigate, and adds friction for readers.

Note that this applies to ALL the README.md files, not just the top level one. Also note that although finding bugs in our code would always be highly rewarded, it is outside the scope of this Bounty, where only the README.md are to be reviewed.

- [community-repo](https://github.com/Joystream/community-repo)

### Reward
Per correction/fix:
- Broken links: USD 3
- Grammatical error`*`: USD 2
- Formatting error: USD 2
Improvements:
- Incorrect information: USD 5 (per fix)
- Expanding or elaborating on incorrect or incomplete information/instructions: 10 USD (per section)
- Adding new documentation`**`: USD 50

- `*` This doesn't include changing between British and US english
- `**` Only for the parts specified in the [Success Event 2](#success-events)

### Format
Jsgenesis suggests a ["free for all"](https://github.com/Joystream/helpdesk/tree/master/roles/builders#free-for-all) format.

Steps for participants:
- Once you have found something, fork the repo, and make the changes on your branch.
- Push your changes, and make a PR with the fixes.
- Request a review in the forum thread(tbd) from the Admin(tbd).
- Request a review from @bwhm or @blrhc.
- Once the PR is merged (this may be an iterative process), create a spending proposal based on the then prevailing exchange and the sum of the merged changes.
- Include a link the PR.

### Success Events
1. For each of the repos below, make a PR improving the docs/guides:
- [helpdesk](https://github.com/Joystream/helpdesk)
- [joystream](https://github.com/Joystream/joystream)
- [hydra](https://github.com/Joystream/hydra)
- [joystream-api-examples](https://github.com/Joystream/joystream-api-examples)
- [atlas](https://github.com/Joystream/atlas)
- [joystream-org](https://github.com/Joystream/joystream-org)

2. Add to the documentation for the following sections of the repos:
- [helpdesk](https://github.com/Joystream/helpdesk)
  - Add a guide for using the CLI for the [`Validator`](https://github.com/Joystream/helpdesk/tree/master/roles/validators) role.
- [joystream-api-examples](https://github.com/Joystream/joystream-api-examples)
  - Update the repo to use the newest types, and make examples for how to use the API to:
    - Fetch historical data from an archival node (proposals and validators)
    - Get information from events (must be combined with historical data)
    - How to get all data from [double maps](https://polkadot.js.org/docs/api/cookbook/storage#how-do-i-use-entrieskeys-on-double-maps)
  - Note that this (outdated and incomplete) [draft PR](https://github.com/Joystream/joystream-api-examples/pull/15) may be of help.

### Conditions
- The fixes are submitted as a PR to the respective repositories
- All fixes are licensed in line with the respective repositories

### Annihilation
Any "fixes" that introduces bugs, anything maliocous or clearly incorrect information.

### Review Period
Once the Council approves the PR, by linking to the PR in this issue, and tagging @bwhm and @blrhc, Jsgenesis will grade and potentially increase the fiat pool, within 72h. If the PR contains code, up to a week must be expected.

### Status
Draft

**Comment by mochet** [comment]
@bwhm / @blrhc can you please add the thread to this issue: https://testnet.joystream.org/#/forum/threads/303

**Comment by mochet** [comment]
This bounty is no longer relevant to the new system, closing this to avoid confusion
